examples/codeviewer/common/src/jvmMain/kotlin/org/jetbrains/codeviewer/platform/JvmFile.kt

```diff
@@ -59,12 +59,15 @@ fun java.io.File.toProjectFile(): File = object : File {
 
             override fun get(index: Int): String {
                 val startPosition = lineStartPositions[index]
-                val length = if (index + 1 < size) lineStartPositions[index + 1] - startPosition else
-                    byteBufferSize - startPosition
+                var endPosition = if (index + 1 < size) lineStartPositions[index + 1] else byteBufferSize
+                endPosition-- // Skip \n
```

Comment:
It fixes the example, but we can have \n and \r in the user code as well, and Compose should behave well if text have them.

Reply:
Discussed, that we have ovelapped lines because of that